node/lib/types.ts

```diff
@@ -12,6 +12,7 @@ export interface ChatJSON {
   color: string
   id: number
   name: string
+  mailinglistAddr: string
```

Comment:
if you mention it here you must also return it from `toJson()` in `node/lib/chat.ts`, but the test also should now fail to tell you this